docs/adr/0021-audit-logging.md

```diff
@@ -0,0 +1,144 @@
+# Audit Logging
+
+**User Story:** [EASi 697](https://jiraent.cms.gov/browse/EASI-697)
+
+As we try to have logs for user actions, we need to better understand where
+they live and what extent of logging we will be doing?
+
+## Decisions / Answers
+
+### 1) Where will the logging live
+
+CMS Splunk Cloudwatch
```

Comment:
```suggestion
CMS Splunk, AWS logging tools (e.g., Cloudwatch, S3)
```
You could include a link to https://github.com/CMSgov/easi-app/blob/master/docs/adr/0009-logging-platform.md here.